src/components/PharmacyCard/PharmacyCard.tsx

```diff
@@ -23,6 +23,7 @@ interface PharmacyProps {
 }
 
 export function PharmacyCard(props: PharmacyProps) {
+  /*
```

Comment:
if `booking` is coming through props, we can probably just delete this rather than comment out